src/transformers/pipelines/document_question_answering.py

```diff
@@ -0,0 +1,459 @@
+import re
```

Comment:
```suggestion
# Copyright 2022 The HuggingFace Team. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

import re
```

Reply:
Is it appropriate to include Impira in the list of copyright here? I noticed a number of other files that were done collaboratively in the repo have multiple copyrights (e.g. "Copyright 2022 University of Wisconsin-Madison and The HuggingFace Inc. team.").

Reply:
cc'ing @Narsil here

Reply:
I have no clue about copyrights in codebase. What should shouldn't be there.

I am not even sure who to ping.

Reply:
Pinging @LysandreJik in that case :)

Reply:
Yes having joint copyright is completely okay for this file.

Reply:
Okay great!